### DISCUSSION TITLE
Portable

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1706] [Ideas]
Hi!
Would it be possible to release a portable version of the browser? I really like the browser but I would like to keep it portable. There already is one portable release of the browser but it's outdated (its version is 92) and no update seems to be coming for it. Would it be possible to build a portable version of the browser that would be released aside the normal version with each update?

**Comment by wchen342** [comment]
Which system? If Linux you can try my script: Eloston/ungoogled-chromium/pull/1693. You will need a Debian based system/docker envirtonment.

If Windows you can try to use this PR: https://github.com/wchen342/ungoogled-chromium-building-scripts/pull/1.

**Reply by wchen342** [reply]
There is also portable builds by third-party: https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/linux_portable/64bit. Note they are not built by maintainers so use with your own judgement.

**Reply by ghost** [reply]
Hi! Thanks for the links. However, I'm not into building the program as it's too complicated for me.
I use Windows.
